scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -1539,6 +1539,22 @@ public void sendRecord(Record record) {
 			public void sendFlight(DTLSFlight flight) {
 				sendHandshakeFlight(flight, connection);
 			}
+
+			@Override
+			public void pauseRetransmission() {
```

Comment:
We cancel the current retransmission task, but not the whole retransmission, I mean we don't reset the flight tries counter and we don't reset the timeout value. That's why I choose `pause`/`resume` instead of `cancel`/`restart`. `restart` make me seem like we restart retransmission from the beginning  (flight tries = 0, timeout = 0).